src/components/NotificationsDrawer/NotificationItem.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import {
+  NotificationDrawerList,
+  NotificationDrawerListItem,
+  NotificationDrawerListItemBody,
+  NotificationDrawerListItemHeader,
+} from '@patternfly/react-core/dist/dynamic/components/NotificationDrawer';
+import { PopoverPosition } from '@patternfly/react-core/dist/dynamic/components/Popover';
+import { Checkbox } from '@patternfly/react-core/dist/dynamic/components/Checkbox';
+import { Label } from '@patternfly/react-core/dist/dynamic/components/Label';
+import {
+  MenuToggle,
+  MenuToggleElement,
+} from '@patternfly/react-core/dist/dynamic/components/MenuToggle';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+} from '@patternfly/react-core/dist/dynamic/components/Dropdown';
+import EllipsisVIcon from '@patternfly/react-icons/dist/dynamic/icons/ellipsis-v-icon';
+import DateFormat from '@redhat-cloud-services/frontend-components/DateFormat';
+import { NotificationData } from '../../state/atoms/notificationDrawerAtom';
+import axios from 'axios';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+interface NotificationItemProps {
+  notification: NotificationData;
+  onNavigateTo: (link: string) => void;
+  updateNotificationSelected: (id: string, selected: boolean) => void;
+  updateNotificationRead: (id: string, read: boolean) => void;
+}
+const NotificationItem: React.FC<NotificationItemProps> = ({
+  notification,
+  onNavigateTo,
+  updateNotificationSelected,
+  updateNotificationRead,
+}) => {
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+
+  const onCheckboxToggle = () => {
+    updateNotificationSelected(notification.id, !notification.selected);
+  };
+
+  const onMarkAsRead = () => {
+    axios
+      .put('/api/notifications/v1/notifications/drawer/read', {
+        notification_ids: [notification.id],
+        read_status: !notification.read,
+      })
+      .then(() => {
+        updateNotificationRead(notification.id, !notification.read);
+        setIsDropdownOpen(false);
+      })
+      .catch((e) => {
+        console.error('failed to update notification read status', e);
+      });
+  };
+
+  const notificationDropdownItems = [
+    <DropdownItem key="read" onClick={onMarkAsRead}>{`Mark as ${
+      !notification.read ? 'read' : 'unread'
+    }`}</DropdownItem>,
+    <DropdownItem
+      key="manage-event"
+      onClick={() =>
+        onNavigateTo(
+          `/settings/notifications/configure-events?bundle=${notification.bundle}&tab=configuration`
```

Comment:
Since we are in the notifications application I think all links needs to be updated and the `/settings` should be removed. I would have to however have to check it locally what needs to remain and what can be removed.

Reply:
Messing around with these links for a minute, the /settings/notifications etc was not prepended. I'm thinking it's because the drawer doesn't go through the router?

Reply:
@karelhala the pathname is not prepended. The drawer is not part of the notifications context. It is "its own app" if you will.